frontend/apps/reader/modules/readerrolling.lua

```diff
@@ -457,6 +468,31 @@ You can set how many lines are shown.]])
         help_text = _([[When page overlap is enabled, some lines from the previous pages are shown on the next page.]]),
         sub_item_table = page_overlap_menu,
     }
+    if self.view.document:hasNonLinearFlows() then
```

Comment:
I know we use `self.view.document` in a few modules - but not in this one, readerrolling.
The canonical way to access the document object is usually with `self.ui.document` (`ui` instead of `view`).

Reply:
At least readerfooter uses both `self.ui.document` and `self.view.document`. Any reason for this or is it OK if I change them all to `self.ui.document` too?